Week 3/shopee-reduxtoolkit-typscript/src/components/Product.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+
+import { Link, useNavigate } from 'react-router-dom';
+import { useAppDispatch } from '../store/store';
+import { addToCart } from "../store/productSlice";
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
```

Comment:
@phamhuutinh0605 : tách css ra từng component
```suggestion
import './Product.scss'
```